src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -54,7 +54,7 @@ The following is a general example of naming convention:
 <pipeline_name>/
     sub-<participant_label>/
         dwi/
-            <source_keywords>[_space-<space>][_desc-<label>]_model-<label>_diffmodel.nii[.gz]
+            <source_keywords>[_space-<space>][_desc-<label>]_model-<label>[_parameter-<parameter>]_diffmodel.nii[.gz]

```

Comment:
I concur. What would we cover with the 'parameter'?

Reply:
In fact I reread and this was related to the type of metric stored (FA, etc).

Reply:
I tried to be as brief as I could initially due to the scope of the PR, but it can come off as cryptic when I try to explain the general case. I'll use the example of NODDI.

-----

With the current spec, one would have:
```
<pipeline_name>/
    sub-<participant_label>/
        dwi/
            sub-<participant_label>_model-NODDI_diffmodel.nii[.gz]
```
This would be a 4D image, with 3 volumes. The user would need to consult the specifications to discover that these three volumes correspond to ICVF, OD, ISOVF in that order.

-----

What I propose would be more intuitive is:
```
<pipeline_name>/
    sub-<participant_label>/
        dwi/
            sub-<participant_label>_model-NODDI_parameter-ICVF_diffmodel.nii[.gz]
            sub-<participant_label>_model-NODDI_parameter-OD_diffmodel.nii[.gz]
            sub-<participant_label>_model-NODDI_parameter-ISOVF_diffmodel.nii[.gz]
```
Each of these files would be a 3D image.

Reply:
Yes sorry I wanted to edit to say I understood later on. Thanks for claryfying for the others as well!

Reply:
This is a good suggestion. Thanks

Reply:
I've given this a go, and modified NODDI and fwDTI accordingly; see what you think. However the AWF parameter in the WMTI model is glaring at me; it's quite clumsy being concatenated onto the end of all the tensor / kurtosis coefficients.